askama_shared/src/generator.rs

```diff
@@ -1819,11 +1825,12 @@ impl<'a, S: std::hash::BuildHasher> Generator<'a, S> {
         self.prepare_ws(ws);
     }
 
-    fn should_trim_ws(&self, ws: Option<Whitespace>) -> bool {
+    fn should_trim_ws(&self, ws: Option<Whitespace>) -> WhitespaceHandling {
         match ws {
-            Some(Whitespace::Trim) => true,
-            Some(Whitespace::Preserve) => false,
-            None => self.whitespace == WhitespaceHandling::Suppress,
+            Some(Whitespace::Trim) => WhitespaceHandling::Suppress,
```

Comment:
We should rename `Whitespace::Trim` to `Suppress`, too.

Reply:
Adding a new commit then. :)